recipe/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
Does 10.7 work?

Also, instead of playing with flags for this more general stuff, it may be worth trying the new [`toolchain`]( https://github.com/conda-forge/toolchain-feedstock ). Take a look at this [commit]( https://github.com/conda-forge/staged-recipes/pull/578/commits/f1f84a51b881555f342c4fcb07f5e0770fbe6a03 ) for an example of how to use it.